### PR TITLE
TagBox: fix issue with Tag Count Limitation Angular Demo (T1017708)

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
             </div>
             <div class="dx-field-value">
                 <dx-tag-box
-                    [items]="products.slice(0, 5)"
+                    [items]="products"
```
Comment (DokaRus) [inline JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.html]:
```suggestion
                    [items]="preparedProducts"
```

File: JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class AppComponent {
     products: Product[];
 
     constructor(service: Service) {
-        this.products = service.getProducts();
+        this.products = service.getProducts().slice(0, 5);
```
Comment (DokaRus) [inline JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts]:
```suggestion
        this.products = service.getProducts();
        this.preparedProducts = this.products.slice(0, 5);
```

File: JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ export class AppComponent {
     products: Product[];
```
Comment (DokaRus) [inline JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts:21]:
```suggestion
    products: Product[];
    preparedProducts: Product[];
```

**Review by DokaRus — DISMISSED** [review]

File: JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts [hunk 4]
```diff
@@ -19,9 +19,11 @@ if(!/localhost/.test(document.location.host)) {
 
 export class AppComponent {
     products: Product[];
+    slicedProducts: Product[];
 
     constructor(service: Service) {
         this.products = service.getProducts();
+        this.slicedProducts = service.getProducts().slice(0, 5);
```
Comment (alexanderPolosatov) [inline JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts]: suggestion:
if replace this to
this.slicedProducts =  this.products.slice(0, 5);
you reduce service calls count.

Comment (Zedwag) [reply JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts]: yep, ty

**Review by alexanderPolosatov — APPROVED** [review]

**Comment by ksercs** [comment]
I think title should be more specific :)

**Review by ksercs — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**